runbook/docs/add-urls-to-monitoring-list-and-database.md

```diff
@@ -0,0 +1,77 @@
+
+Adding new URLs involves 2 main steps:
+
+* Add URLs Internet Archive Wayback Machine
+* Add URLs to Scanner's database
+
+## Prerequsites
+- Have a clone of [web-monitoring-ops](https://github.com/edgi-govdata-archiving/web-monitoring-ops) repository
+
+## Steps
+
+### **Add URLs to Internet Archive Wayback Machine**
+- Clone [wayback-spn-client](https://github.com/Mr0grog/wayback-spn-client)
+- Create a `.txt` file with the list of new URLs to be tracked
+- From `wayback-spn-client` repo: 
+    - `> node index.js path/to/your/url/list.txt`
+        - This process takes about 30sec per url
+        - Make sure output doesn't contain errors
+
+From `web-monitoring-ops` repo:
+- Add URLs to `web-monitoring-ops/manually-managed/ia-archiver/missing-from-ia.txt`
+    - Add a comment describing why the URLs were added
+- Commit `missing-from-ia.txt` back to the repo.
+    - In general, we don't use PRs for updating the `missing-from-ia.txt`. Commit to `master` if you're comfortable doing so.
+- Copy to `ia-archiver` server from `web-monitoring-ops`
+    ```
+    > scp manually-managed/ia-archiver/missing-from-ia.txt ubuntu@<ipaddress>:/home/ubuntu/
+    ```
+- Inform Internet Archive about added URLs in the Scanner slack channel. 
+    - Usually @mr0grog does this to keep our point-of-contact with the Internet Archive consistent
+
+### **Adding URLs to Scanner's database**
+* Clone [web-monitoring-processing](https://github.com/edgi-govdata-archiving/web-monitoring-processing)
+* Be sure to follow all installation instructions carefully
+* You will need to have write permissions and credentials to our database. Add those to `.env` file.
+* For each URL:
+    - `> wm import ia <URL> --tag "tag" --maintainer "maintainer"`
+        - Be sure to add a `tag` and `maintainer`. You can use the [API explorer](https://api.monitoring.envirodatagov.org/) to look for examples from related, existing records
+
+* Watch for a job import ID, so you know the import was successful. Also, can double check at [api.monitoring.envirodatagov.org](api.monitoring.envirodatagov.org)
+
+* OPTIONAL: If URLs were not able to be imported to the Wayback Machine via SPN (Save Page Now), you will have to add the pages to the database manually instead of using `wm import`
+    1. Add a page record to the DB:
+
+        1. SSH into a live server via Kubernetes:
+
+            ```sh
+            # List available pods
+            > kubectl get pods --namespace production
+            # Pick an `api-` or `import-worker-` pod (basically, one of the DB pods) and log in
+            > kubectl exec -it <name of pod> /bin/bash --namespace production
```

Comment:
This is out of date! It still works, but this form of the command is deprecated. It should now be:

```suggestion
            > kubectl exec -it --namespace production <name of pod> -- /bin/bash
```